public/static/docs/api-reference/read.md

```diff
@@ -0,0 +1,88 @@
+# dvc.api.read()
+
+Returns the contents of a tracked file.
+
+## Usage
+
+```py
+import dvc.api
+
+modelpkl = dvc.api.read(
+    'model.pkl',
+    repo='https://github.com/example/project.git'
+    mode='rb')
+```
+
+### Signature
+
+```py
+read(path, repo=None, rev=None, remote=None, mode="r", encoding=None)
+```
+
+### Types
+
+All **parameter** types are
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+The **return** type can be a
+[bytes object](https://docs.python.org/3/glossary.html#term-bytes-like-object)
+or a
+[string](https://docs.python.org/3/library/stdtypes.html#text-sequence-type-str).
+
+No exceptions are raised by this function.
+
+## Description
+
+This function wraps [`dvc.api.open()`](/doc/api-reference/open) for a simple and
+direct way to return the complete contents of files tracked in <abbr>DVC
+projects</abbr> (by DVC or Git) – no _context manager_ (`with` keyword) needed.
```

Comment:
same - context managers - we need to review their usage (see other comments about them)

Reply:
This has already been removed from this doc.